.pre-commit-config.yaml

```
@@ -189,6 +189,11 @@ repos:
     additional_dependencies:
     - lxml  # dep of `--txt-report`, `--cobertura-xml-report` & `--html-report`
     - pytest
+    # The following dependency is used in `credentials.aim`,
+    # `credentials.centrify_vault`, `credentials.conjur`
+    # and `credentials.hashivault`:
+    - types-requests  # dep of `credentials.aim`, `credentials.centrify_vault`,
+    # `credentials.conjur` & `credentials.hashivault`
```

No need to repeat the above comment once more.
```suggestion
    - types-requests
```